cyberdrop_dl/crawlers/anysex.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, ClassVar
+
+from cyberdrop_dl.crawlers._fluid_player import FluidPlayerCrawler
+from cyberdrop_dl.data_structures.url_objects import AbsoluteHttpURL
+from cyberdrop_dl.utils import css, open_graph
+from cyberdrop_dl.utils.utilities import error_handling_wrapper
+
+if TYPE_CHECKING:
+    from cyberdrop_dl.crawlers.crawler import SupportedPaths
+    from cyberdrop_dl.data_structures.url_objects import ScrapeItem
+
+
+class Selector:
+    ALBUM_NAME = "h1.title"
+    SWIPE_WRAPPER = "div.swiper-wrapper div.swiper-slide a"
+    MAIN_IMAGE_HOLDER = "a#main_image_holder"
```

Comment:
Combine this 2 selectors into 1
```suggestion
    IMAGES = "a#main_image_holder, .swiper-wrapper .swiper-slide a"
```